apps/nrm/translations/src/en/fields.json

```diff
@@ -547,17 +547,19 @@
     "paragraph-1": "If a potential victim wants you to refer their case, they can get:",
     "bullet-1": "access to support (for example, housing and legal advice)",
     "bullet-2": "a formal decision – this can help if they’re being prosecuted for a crime they were forced to commit",
-    "div-1-paragraph-1": "We will need to share their personal details with organisations such as the police and the Home Office if they agree to their case being referred.",
-    "div-1-paragraph-2": "Please explain to the potential victim the ways in which the Home Office may process their personal information, including the purposes for which it will be used, the legal basis and who it will be shared with. This is contained in the <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/human-trafficking-victims-referral-and-assessment-forms/privacy-information-notice-national-referral-mechanism\">Privacy Information Notice</a> and <a class=\"govuk-link\" href=\"https://www.gov.uk/government/organisations/home-office/about/personal-information-charter\">Personal Information Charter.</a>",
+    "paragraph-2": "You should:",
+    "div-1-paragraph-1": "make sure that the potential victim understands by entering the NRM they are consenting to a reasonable grounds and conclusive grounds decision being made",
+    "div-1-paragraph-2": "make sure that the potential victim understands the support that they may be able to receive, as detailed in the <a class=\"govuk-link\" href=\"https://assets.publishing.service.gov.uk/media/67812b3b00e3d719f19217eb/Modern+Slavery+Statutory+Guidance+_EW_+and+Non-Statutory+Guidance+_SNI_+v3.12.pdf#page=65\">guidance (see section 'Support for Adult Victims').</a> This may require the assistance of an appropriate interpreter.",
+    "div-1-paragraph-3": "explain to the potential victim the ways in which the Home Office may process their personal information, including the purposes for which it will be used, the legal basis and who it will be shared with. You can find this information in the This is contained in the <a class=\"govuk-link\" href=\"https://www.gov.uk/government/publications/human-trafficking-victims-referral-and-assessment-forms/privacy-information-notice-national-referral-mechanism\">Privacy Information Notice</a> and <a class=\"govuk-link\" href=\"https://www.gov.uk/government/organisations/home-office/about/personal-information-charter\">Personal Information Charter.</a>",
```

Comment:
I thought I added this point earlier but I might not have saved it - since these are now list items instead of paragraphs it might be better to rename the properties to something like `bullet-3 `or `list-item-1`, etc